src/JetBrains/BenchmarkDotNet.Diagnostics.dotMemory/BenchmarkDotNet.Diagnostics.dotMemory.csproj

```
@@ -10,7 +10,11 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <ProjectReference Include="..\BenchmarkDotNet\BenchmarkDotNet.csproj" />
+        <ProjectReference Include="..\..\BenchmarkDotNet\BenchmarkDotNet.csproj" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <Compile Include="..\Shared\*.cs" />
```

Recursively includes items in sub-folders (just in case for future).

```suggestion
        <Compile Include="..\Shared\**/*.cs" />
```